pyb/pyb.py

```diff
@@ -9,13 +9,12 @@ def on(self):
     def off(self):
         self.f.write(b"0")
 
+    def set(self, v):
+        self.f.write(b"{}".format(0 if v < 0 else 255 if v > 255 else v))
```

Comment:
Maybe, but it involves 2 function calls. And the original is almost plain English anyway ;-).